commands/show-beacons.go

```diff
@@ -33,28 +32,31 @@ func showBeacons(c *cli.Context) error {
 	if c.String("database") == "" {
 		return cli.NewExitError("Specify a database with -d", -1)
 	}
-	res := database.InitResources("")
+	res := database.InitResources(c.String("config"))
 	res.DB.SelectDB(c.String("database"))
 
 	var data []beaconData.BeaconAnalysisView
-	cutoffScore := .7
-	if c.Bool("all") {
-		cutoffScore = 0
-	}
 
 	ssn := res.DB.Session.Copy()
-	resultsView := beacon.GetBeaconResultsView(res, ssn, cutoffScore)
+	resultsView := beacon.GetBeaconResultsView(res, ssn, 0)
```

Comment:
To me, replacing `cutoffScore` with `0` makes this harder to read.  I understand why a variable is no longer needed, but it's a shame to remove the symbolism.  I'd suggest either adding a comment to explain the 0 value or keeping the variable for readability's sake.